Userland/Libraries/LibC/fenv.h

```diff
@@ -32,6 +34,42 @@ typedef struct fenv_t {
     uint32_t __mxcsr;
 } fenv_t;
 
+#elif defined(__aarch64__)
+
+// TODO: Implement this.
+typedef struct fenv_t {
+} fenv_t;
+
+#elif defined(__riscv) && __riscv_xlen == 64
+
+// Chapter numbers from RISC-V Unprivileged ISA V20191213
+// RISC-V F extension version 2.2, Figure 11.1
+typedef struct fenv_t {
+    union {
+        // 11.2: fcsr is always 32 bits, even for the D and Q extensions, since only the lowest byte of data is in use.
+        uint32_t fcsr;
+        [[gnu::packed]] struct {
+            // Accrued exceptions (fflags).
+            union {
+                uint8_t accrued_exceptions : 4;
+                [[gnu::packed]] struct {
+                    uint8_t inexact : 1;           // NX
+                    uint8_t underflow : 1;         // UF
+                    uint8_t overflow : 1;          // OF
+                    uint8_t divide_by_zero : 1;    // DZ
+                    uint8_t invalid_operation : 1; // NV
+                };
+            };
+            uint8_t rounding_mode : 3; // frm
+            uint32_t reserved : 24;
+        };
```

Comment:
is this beauty worth an AssertSize?

Reply:
Yes, 'cause it is not even correct.

Reply:
(And my -Wpadded-bitfield actually triggers on it!)